src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -58,6 +58,29 @@ public void SetEncoding(bool value)
         int IVsEditorFactory.Close()
             => VSConstants.S_OK;
 
+        public int ChooseEditorFactory(
+            string pszMkDocument,
+            IVsHierarchy pHier,
+            uint itemid,
+            IntPtr punkDocDataExisting,
+            ref Guid rguidLogicalView,
+            out Guid pguidEditorTypeActual,
+            out Guid pguidLogicalViewActual)
+        {
+            if (rguidLogicalView == VSConstants.LOGVIEWID.Designer_guid)
+            {
+                pguidEditorTypeActual = typeof(IWinFormsEditorFactory).GUID;
```

Comment:
[IWinFormsEditorFactory](https://devdiv.visualstudio.com/DevDiv/_git/VS?path=%2Fsrc%2Fvsip%2FPackages%2FInterfaces%2FPublic%2FIWinFormsEditorFactory.cs&_a=contents&version=GBmain) type is not derived from `IVsEditorFactory` and hence it is not an actual editor factory recognized by VS. It is only a service proffered by WinForms designer package. Thus, this change will not work as intended.
To make it work, we will first need to proffer an editor factory from WinForms package and then specify it here.
 
I will make that change in WinForms code and then create a PR here, if required.

cc: @rafaci, @merriemcgaw